### PR TITLE
fix quantile_exact parameter error

### PR DESCRIPTION
**Pull request by lizhou1111**
PR checklist:
- Did you run ClangFormat ?
- Did you separate headers to a different section in existing community code base ?
- Did you surround `proton: starts/ends` for new code in existing community code base ?

Please write user-readable short description of the changes:

### CODE REVIEW

**Comment by lizhou1111** [comment]
this close #725

**Comment by yokofly** [comment]
add a test? either in stateless or in smoke.

File: src/Interpreters/ExpressionAnalyzer.cpp [hunk 1]
```diff
@@ -174,28 +174,38 @@ void tryTranslateToParametricAggregateFunction(
     }
     else if (lower_name.starts_with("quantile") && !lower_name.ends_with("if"))
```
Comment (yl-lisen) [inline src/Interpreters/ExpressionAnalyzer.cpp]: so far, we need block all aggregate functions with suffix `AggregateFunctionCombinatorFactory::tryFindSuffix`

File: tests/queries_ported/0_stateless/99002_function_quantile.sql [hunk 2]
```diff
@@ -3,3 +3,34 @@ with cte as (select array_join([100, 200, 300, 400, 500]) as f) select quantile(
...
+SELECT quantiles_timing(x, 1, 0.001, 0.01, 0.05, 0.9, 0.2, 0.3, 0.6, 0.5, 0.4, 0.7, 0.8, 0.1, 0.95, 0.99, 0.999, 0, 0.5, 0.3, 0.4) FROM (SELECT number AS x FROM system.numbers LIMIT 1001);
+SELECT quantiles(x, 1, 0.001, 0.01, 0.05, 0.9, 0.2, 0.3, 0.6, 0.5, 0.4, 0.7, 0.8, 0.1, 0.95, 0.99, 0.999, 0, 0.5, 0.3, 0.4) FROM (SELECT number AS x FROM system.numbers LIMIT 1001);
+SELECT quantiles_deterministic(x, x, 1, 0.001, 0.01, 0.05, 0.9, 0.2, 0.3, 0.6, 0.5, 0.4, 0.7, 0.8, 0.1, 0.95, 0.99, 0.999, 0, 0.5, 0.3, 0.4) FROM (SELECT number AS x FROM system.numbers LIMIT 1001);
+
+SELECT quantile_exact_weighted(x, 1, 0.5) AS q5, quantiles_exact_weighted(x, 1, 0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1) AS qs FROM (SELECT array_join([1, 1, 1, 10, 10, 10, 10, 100, 100, 100]) AS x);
+SELECT quantile_exact(x, 0), quantile_timing(x, 0) FROM (SELECT number + 100 AS x FROM system.numbers LIMIT 10000);
+SELECT quantile_exact(x), quantile_timing(x) FROM (SELECT number % 123 AS x FROM system.numbers LIMIT 10000);
```
Comment (yl-lisen) [inline tests/queries_ported/0_stateless/99002_function_quantile.sql]: end with an empty line

**Review by yl-lisen — APPROVED** [review]

**Pull request merged**